config/devel.webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const { resolve } = require('path');
+
+const config = require('@redhat-cloud-services/frontend-components-config');
+
+const { config: webpackConfig, plugins } = config({
+  rootFolder: resolve(__dirname, '../'),
+  debug: true,
+  useFileHash: false,
+  sassPrefix: '.imageBuilder',
+  deployment: 'beta/apps',
+  appUrl: '/preview/insights/image-builder',
+  env: 'prod-beta',
+  useProxy: true,
+  useAgent: false,
```

Comment:
```suggestion
  useAgent: true,
  ```

Reply:
If we want to use the `stage` environment, we need to also set `useAgent` to `true`.